JS/Lesson34/gulcin/recipe-app/src/App.js

```diff
@@ -1,23 +1,21 @@
 import { useState, useEffect } from 'react';
 import { Search } from './modules/recipes/SearchBar';
+import { RecipeDetails } from './modules/recipes/RecipeDetails';
 import './App.css';
+import { NewRecipe } from './modules/recipes/NewRecipe';
 
 export const App = () => {
 	const [recipes, setRecipes] = useState([]);
-
-	useEffect(() => {
```

Comment:
You still need to fetch recipes, don't remove it. It's just that everything is on the same page for this assignment but it will be fixed in the Lesson 37